recipes/wincertstore/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "wincertstore" %}
+{% set version = "0.2" %}
+
+package:
+  name: python-{{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  md5: ae728f2f007185648d0c7a8679b361e2
+
+build:
+  number: 0
+  skip: true  # [not win]
```

Comment:
This is only for Windows as it is designed to work with Windows’s sytem cert store.